src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -54,6 +54,8 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true'">
     <Compile Include="System\Net\NameResolutionPal.Windows.cs" />
+    <Compile Include="System\Net\NameResolutionPal.Windows.NetCoreApp.cs" Condition="'$(TargetGroup)' != 'uap'"/>
```

Comment:
We typically use Win32.cs suffix for these.

Reply:
So .Win32.cs and .Uap.cs suffixes then?

Reply:
I think so. Here is example of prior art:
```
ProcessStartInfo.Uap.cs
ProcessStartInfo.Unix.cs
ProcessStartInfo.Win32.cs
ProcessStartInfo.Windows.cs
```